lib/node_modules/@stdlib/math/base/special/cpolar/docs/repl.txt

```diff
@@ -1,35 +1,54 @@
 
-{{alias}}( [out,] re, im )
-    Returns the absolute value and phase of a complex number.
+{{alias}}( z )
+    Returns the absolute value and phase of a double-precision complex
+    floating-point number.
 
     Parameters
     ----------
-    out: Array|TypedArray|Object (optional)
-        Output array.
-
-    re: number
-        Real component.
-
-    im: number
-        Imaginary component.
+    z: Complex128
+        Complex number.
 
     Returns
     -------
-    out: Array|TypedArray|Object
+    out: Array
         Absolute value and phase, respectively.
 
     Examples
     --------
-    > var out = {{alias}}( 5.0, 3.0 )
+    > var out = {{alias}}( new {{alias:@stdlib/complex/float64}}( 5.0, 3.0 ) )
     [ ~5.83, ~0.5404 ]
 
-    // Provide an output array:
-    > out = new {{alias:@stdlib/array/float64}}( 2 );
-    > var v = {{alias}}( out, 5.0, 3.0 )
+
+{{alias}}.assign( z, out, stride, offset )
+    Returns the absolute value and phase of a double-precision complex
+    floating-point number and assigns results to a provided output array.
+
+    Parameters
+    ----------
+    z: Complex128
+        Complex number.
+
+    out: Array<number>
+        Destination array.
+
+    stride: integer
+        Output array stride.
+
+    offset: integer
+        Output array index offset.
+
+    Returns
+    -------
+    out: Array<number>
```

Comment:
```suggestion
    out: Array|TypedArray|Object
```

The `out` argument can be any array-like object and the elements need not all be numeric.